pgt/src/PGT/Output/examples/query-plan.sql.expected

```diff
@@ -0,0 +1,69 @@
+       Column        |                Type                | Collation | Nullable | Default
+---------------------+------------------------------------+-----------+----------+---------
+ id                  | uuid_not_null                      |           |          |
+ created_at          | timestamptz_not_null               |           |          |
+
+-- Example comment 1
+-- (1 row)
+
+-[ RECORD 1 ]---------------+------------------------------------------------------------
+id               | 00000000-0000-7000-0000-000000000137
+created_at       | 1970-01-02 02:02:10
+
+-- Example comment 2
+-- another sub-comment
+-- (2 row)
+
+-[ RECORD 1 ]---------------+------------------------------------------------------------
+id               | 00000000-0000-7000-0000-000000000137
+created_at       | 1970-01-02 02:02:10
+-[ RECORD 2 ]---------------+------------------------------------------------------------
+id               | 00000000-0000-7000-0000-000000000138
+created_at       | 1970-01-02 02:02:11
+
+             QUERY PLAN
+-------------------------------------
+ [                                  +
+   {                                +
+     "Plan": {                      +
+       "Node Type": "Result",       +
+       "Parallel Aware": false,     +
+       "Startup Cost": 0.00,        +
+       "Total Cost": 0.01,          +
+       "Plan Rows": 1,              +
+       "Plan Width": 4,             +
+       "Actual Startup Time": 0.001,+
```

Comment:
I do not think this will be stable. We probably want to filter it out in golden tests.